packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -0,0 +1,713 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'framework.dart';
+import 'scroll_controller.dart';
+import 'scroll_physics.dart';
+import 'scrollable.dart';
+
+
+/// A box in which widgets on a wheel can be scrolled.
+///
+/// This widget is similar to a [ListView] but with the restriction that all
+/// children must be the same size along the scrolling axis.
+///
+/// The children starts at the middle instead of at the top of the viewport.
+///
+/// Scrolling also ends with the last child at the middle instead of at the
+/// bottom of the viewport.
+///
+/// The children are also rendered as if on a wheel instead of on a plane.
```

Comment:
Try to avoid the second "also". Maybe just:

```
When the list is at the zero scroll offset, the first child is aligned with the middle of the viewport. When the list is at the final scroll offset, the last child is aligned with the middle of the viewport. The children are rendered as if on a wheel whose axis of rotation is laid horizontal across the coordinate system (as opposed to scrolling on a plane).
```

Reply:
Done